src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Artifactory/ServerlessArtifactoryRepository.tsx

```diff
@@ -79,7 +78,7 @@ export default function ServerlessArtifactoryRepository(
         erroObj?.fieldId && erroObj?.error ? `${erroObj?.fieldId} ${erroObj?.error}` : getString('somethingWentWrong')
       setConnectorRepos([{ label: errorMessage, value: errorMessage }])
     }
-  }, [artifactRepoLoading, artifactRepoError])
+  }, [artifactRepoLoading])
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has missing dependencies: 'artifactRepoError?.data' and 'getString'. Either include them or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232287)
```suggestion
  }, [artifactRepoError?.data, artifactRepoLoading, getString])
```